libqtile/widget/base.py

```diff
@@ -481,15 +533,11 @@ def draw(self):
         if not self.can_draw():
             return
         self.drawer.clear(self.background or self.bar.background)
-        if self.bar.horizontal:
-            self.layout.draw(
-                self.actual_padding or 0,
-                int(self.bar.height / 2.0 - self.layout.height / 2.0) + 1,
-            )
-        else:
-            # We need to do some transformations for vertical bars.
-            self.drawer.ctx.save()
 
+        size = self.bar.height if self.bar.horizontal else self.bar.width
```

Comment:
```suggestion
        size = self.bar.size
```
Looks like `size` is only used in place so may as well use `self.bar.size` inline.